tools/Experiments/src/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.

```

Comment:
Headers changed, if you merged main this should helpfully cause an error now. C++ files won't cause an error, but those will need to be addressed as well.